templates/aws/pro.html

```diff
@@ -0,0 +1,267 @@
+            </tr>
+            <tr>
+              <td>Maintenance commitment</td>
+              <td>5 years</td>
+              <td>10 years</td>
+            </tr>
+          </tbody>
+        </table>
+        <p><a href="https://assets.ubuntu.com/v1/579f0c03-Ubuntu_Pro_AWS.pdf" class="p-link--external">Read the Ubuntu Pro datasheet</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light u-no-padding--top">
+  <div class="row">
+    <div class="col-8">
+      <h2>Pricing for US East (N. Virginia) Region</h2>
+      <div style="overflow-x: auto;">
+        <table class="p-table" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Example instance size</th>
+              <th class="u-align--right">EC2 hourly</th>
+              <th class="u-align--right">Ubuntu Pro hourly</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>t3.nano</td>
+              <td class="u-align--right">$0.005</td>
+              <td class="u-align--right">$0.001</td>
+            </tr>
+            <tr>
+              <td>t3.medium</td>
+              <td class="u-align--right">$0.042</td>
+              <td class="u-align--right">$0.009</td>
+            </tr>
+            <tr>
+              <td>c5.large</td>
+              <td class="u-align--right">$0.085</td>
+              <td class="u-align--right">$0.019</td>
+            </tr>
+            <tr>
+              <td>r5.xlarge</td>
+              <td class="u-align--right">$0.252</td>
+              <td class="u-align--right">$0.046</td>
+            </tr>
+            <tr>
+              <td>i3.2xlarge</td>
+              <td class="u-align--right">$0.624</td>
+              <td class="u-align--right">$0.088</td>
+            </tr>
+            <tr>
+              <td>r5d.4xlarge.nano</td>
+              <td class="u-align--right">$1.152</td>
+              <td class="u-align--right">$0.127</td>
+            </tr>
+          </tbody>
+        </table>
+        <p>For complete pricing, <a href="https://aws.amazon.com/marketplace/pp/B0821T9RL2" class="p-link--external">see Ubuntu Pro 18.04 LTS on the AWS Marketplace</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <h2>Get Ubuntu Pro on AWS now:</h2>
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821T9RL2" class="p-link--external">18.04 LTS</a></h3>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821WW873" class="p-link--external">16.04 LTS</a></h3>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821V7QJP" class="p-link--external">14.04 LTS</a></h3>
+    </div>
+  </div>
+  <div class="row">
```

Comment:
```suggestion
  <div class="u-fixed-width">
```

Reply:
I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.